json-csharp/GetValueByKeyFromJObject/GetValueByKeyFromJObject/GetValueByKeyFromJObject.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>disable</Nullable>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <LangVersion>preview</LangVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <LangVersion>preview</LangVersion>
+  </PropertyGroup>
```

Comment:
Same as above.

Reply:
These lines are here because I'm using raw string literals in my source code, and these literals are a preview feature in .NET 7. If I remove these lines, the source code fails to compile.

Reply:
I've tried it locally, and it seems to compile and work properly. 
Are you sure you have the latest version of .Net7 installed in your machine?

Reply:
I've installed the latest version and updated the PR as well. It seems that was the issue.